File: python/set08/problems/Problem2.py
# Create a function that will return a Boolean value indicating if two circles defined by center coordinates and radius are intersecting

x1 , y1 = input("Enter the coordinates of the center of the first circle: ").split(",")
x2 , y2 = input("Enter the coordinates of the center of the second circle: ").split(",")
r1 = int(input("Enter the radius of first circle: "))
r2 = int(input("Enter the radius of the second circle: "))
distance = ((int(x2)-int(x1))**2 + (int(y2)-int(y1))**2)**0.5

if distance < (r1+r2):
    print("True")
else:
    print("False")

File: javascript/set08/Problems/Problem2.js
// Create a function that will return a Boolean value indicating if two circles defined by center coordinates and radius are intersecting

let x1 = prompt("Enter x coordinate of the first center:");
let y1 = prompt("Enter y coordinate of the first center:");
let x2 = prompt("Enter x coordinate of the second center:");
let y2 = prompt("Enter y coordinate of the second center:");
let r1 = prompt("Enter the radius of first circle: ");
let r2 = prompt("Enter the radius of the second circle: ");
let distance = Math.sqrt((x2-x1)**2 + ((y2)-(y1))**2);

let intersecting= distance <= (r1+r2);
console.log(intersecting);
